modules/configuring-localnet-switched-topology.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-secondary-networks.adoc
+
+:_content-type: CONCEPT
+[id="configuration-localnet-switched-topology_{context}"]
+= Configuration for a localnet switched topology
+
+The switched (localnet) topology interconnects the workloads through a cluster-wide logical switch to a physical network.
+
+The following `NetworkAttachmentDefinition` custom resource definition (CRD) YAML describes the fields needed to configure a localnet secondary network.
+
+[source,yaml]
+----
+    {
+            "cniVersion": "0.3.1",
+            "name": "ns1-localnet-network",
+            "type": "ovn-k8s-cni-overlay",
+            "topology":"localnet",
```

Comment:
For 4.13, the code does not support "topology":"localnet"

Reply:
See note above for similar response.